SingularityBase/src/main/java/com/hubspot/singularity/SingularityDeploy.java

```diff
@@ -134,6 +139,9 @@ public SingularityDeploy(@JsonProperty("requestId") String requestId,
     this.serviceBasePath = serviceBasePath;
     this.loadBalancerGroups = loadBalancerGroups;
     this.loadBalancerOptions = loadBalancerOptions;
+
+    this.deployRate = deployRate;
```

Comment:
kind of a nitpick but `deployRate` seems a little confusing to me. maybe `deployInstanceCountPerStep`? (curious what @wsorenson thinks before you make the change though)